src/tracing/errors.ts

```diff
@@ -40,6 +40,17 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
     // If the error is based on a user rejecting, it should not be considered an exception.
     if (didUserReject(error)) return null
 
+    if (error.message.match(/Loading chunk \d+ failed\. \(error: .+\.chunk\.js\)/)) {
```

Comment:
nit: could you preface this `if` condition with a comment explaining why this error happens and why it's okay to filter out `499`? Is it false positive on Cloudflare, but application still works OK?

Reply:
yes i'll do